completions/reportbug

```diff
@@ -27,41 +27,19 @@ _reportbug()
                 done)' -- "$cur"))
             return
             ;;
-        --bts | -!(-*)B)
-            COMPREPLY=($(compgen -W "debian guug kde mandrake help" -- \
-                "$cur"))
+        --tag | --ui | --interface | --type | --bts | --severity | --mode | -!(-*)[TutBS])
+            COMPREPLY+=($(
+                compgen -W \
+                    '$("$1" $prev help 2>&1 | sed -ne /^[[:space:]]/p)' \
```

Comment:
Thank you for checking the behavior. I have installed `reportbug` in my virtual machine and have checked the behavior. I think we can explicitly exclude a case like `prev=-utext` (which is interpreted as `-u text`), though `reportbug -utext help` seems to anyway fail for this particular case.

Reply:
I tried `result=$(reportbug -utext help)` in Debian, but it seems to try to start an interactive session for a bug report. I now believe we should exclude the cases that `prev` is an option flag plus an option argument like `-utext` (`-u` + `text`) as suggested in https://github.com/scop/bash-completion/pull/852#discussion_r1041399560.

Note: In the previous reply, I actually tried `reportbug` in Ubuntu. I reported that `reportbug -utext help` fails anyway, but it was just because `reportbug` didn't support Ubuntu.